pkg/azuredisk/controllerserver.go

```diff
@@ -139,16 +139,20 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	// normalize values
 	skuName, err := azureutils.NormalizeStorageAccountType(diskParams.AccountType, localCloud.Config.Cloud, localCloud.Config.DisableAzureStackCloud)
 	if err != nil {
-		return nil, err
+		return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
Do we need to add this change to V2 as well, here: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/b000a11c697a32dc80a82e204329cdeac166693d/pkg/provisioner/cloudprovisioner.go#L158

Reply:
nvm it was added.